examples/user-guides/configuring-prometheus-alertmanager/alertmanager-config-frontend-receiver.yaml

```diff
@@ -19,6 +19,8 @@ route:
 receivers:
 - name: default
 - name: deadmansswitch
+  webhook_configs:
```

Comment:
This file is used for templating the section right before the DeadMansSnitch configuration section. Could you just create two versions of the files: one with, and one without the DeadMansSnitch configuration. Then we can embed both of them in the documentation.

Reply:
What would you like to call it?

Reply:
let's just call it the same but with `-deadmanssnitch.yaml` suffix